build-system/tasks/storybook-react/README.md

```diff
@@ -0,0 +1,222 @@
+# Storybook
+
+Task that launches [Storybook](https://storybook.js.org/) for isolated component testing.
+
+```console
+amp storybook-react
+```
+
+## Style Recommendations
+
+Follow these recommendations when [writing stories.](https://storybook.js.org/docs/guides/guide-preact/#step-4-write-your-stories)
+
+### Do not include `Bento` in Story titles
+
+Even though a component's name may include `Bento`, it should be excluded from the title for consistency with existing stories.
+
+✅ Do:
+
+```jsx
+export default {
+  title: 'Accordion',
+  component: BentoAccordion,
+};
+```
+
+🚫 Don't:
+
+```jsx
+export default {
+  title: 'BentoAccordion',
+  component: BentoAccordion,
+};
+```
+
+### Identify `args` and `argTypes` exactly like their component properties
+
+Passing a value from a Control specified through `args` or `argTypes` without modification is implicit when spreading `{...args}`.
+
+It's preferred to name a Control property id uniformly with its corresponding component property or element attribute. This way, we write a shorter and more generic story by taking advantage of `{...args}`:
+
+✅ Do:
+
+```js
+const MyStory = (args) => {
+  return <amp-my-component {...args} />;
+};
+
+MyStory.args = {
+  width: 500,
+  height: 200,
+  layout: 'responsive'
+  'data-foo': 'bar',
```

Comment:
Question for myself, which may lead to adding docs. Would this have the same effect?
```jsx
MyStory.args = {
  width: 500,
  height: 200,
  layout: 'responsive',
  data: { foo: 'bar' },
};
```

Reply:
I feel like we alias this amp attributes to preact props, so I don't think it'd map that smoothly. But I'm not clear on how all this all works under the hood.

Reply:
@dethstrobe That's correct